Drop after lang/rust >= 1.76 update

error[E0658]: cannot cast `dyn ai::Action<S, R>` to `dyn std::any::Any`, trait upcasting coercion is experimental
   --> rtsim/src/ai/mod.rs:114:16
    |
114 |         match (other as &dyn Any).downcast_ref::<Self>() {
    |                ^^^^^
    |
error[E0658]: cannot cast `dyn ai::Action<S, R>` to `dyn std::any::Any`, trait upcasting coercion is experimental
   --> rtsim/src/ai/mod.rs:339:16
    |
339 |         match (other as &dyn Any).downcast_ref::<Self>() {
    |                ^^^^^
    |
    = note: see issue #65991 <https://github.com/rust-lang/rust/issues/65991> for more information
    = help: add `#![feature(trait_upcasting)]` to the crate attributes to enable
    = note: required when coercing `&(dyn ai::Action<S, R> + 'static)` into `&(dyn std::any::Any + 'static)`

--- rtsim/src/lib.rs.orig	2024-02-07 19:13:27 UTC
+++ rtsim/src/lib.rs
@@ -2,6 +2,7 @@
     never_type,
     try_blocks,
     trait_alias,
+    trait_upcasting,
     control_flow_enum,
     let_chains,
     binary_heap_drain_sorted,
